player/player.gd

```diff
@@ -140,3 +135,8 @@ func _on_player_body_enter_shape( body_id, body, body_shape, local_shape ):
 func _on_player_body_exit_shape( body_id, body, body_shape, local_shape ):
 	if( local_shape == 1 ):
 		is_grounded = false
+
+func _input(event):
```

Comment:
I would personally prefer this above `_fixed_process`, but I'm fine with it staying here too...